src/cursor/cur_hs.c

```diff
@@ -528,6 +528,21 @@ __curhs_next_visible(WT_SESSION_IMPL *session, WT_CURSOR_HS *hs_cursor)
     return (ret);
 }
 
+/*
+ * __wt_hs_upd_time_window --
+ *     Get the underlying time window of the hs cursor.
+ */
+void
+__wt_hs_upd_time_window(WT_CURSOR *hs_cursor, WT_TIME_WINDOW **twp)
```

Comment:
Could we move this to other history store file.

Reply:
Which one you are thinking of? I think this file should be all the hs api live.

Reply:
I am thinking of either hs_cursor.c or hs_rec.c

Reply:
OK, I'll move it to hs_cursor.c